packages/itwin/property-grid/src/test/setup.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+// Node 15+ using MessageChannel prevents node.js process from exiting
+// This becomes an issue when testing React code within JSDOM environment, as the test process cannot exit properly.
+// https://github.com/facebook/react/issues/20756
+const commonjsGlobal: { MessageChannel?: any } = typeof globalThis !== "undefined" ? globalThis : typeof window !== "undefined" ? window : typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : {};
+if (commonjsGlobal.MessageChannel)
+  delete commonjsGlobal.MessageChannel;
+
+import jsdomGlobal from "jsdom-global";
+import * as chai from "chai";
+import sinonChai from "sinon-chai";
+
+jsdomGlobal();
+
+// setup chai
+chai.should();
```

Comment:
are we using the "should" style anywhere?

Reply:
No. Removed this line